### PR TITLE
feat: design changes - ACC-143

### PR DESCRIPTION
**Pull request by MarcinR**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-143" title="ACC-143" target="_blank">StoryACC-143</a>  Context change on focus (3.2.1)
  </td></table>
  <br />
 

ACC-380 ACC-400
New incoming call view on horizontal.
Fix avatar-label spacing on group call with > 6 participants

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

### CODE REVIEW

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1823,7 +1822,6 @@
 			dstSubfolderSpec = 10;
 			files = (
 				A96867AD26A5E59F00679D95 /* WireDataModel.xcframework in Embed Frameworks */,
-				01AAD75D294FD4CF00F1CFFD /* Datadog.xcframework in Embed Frameworks */,
```
Comment (KaterinaWire) [inline Wire-iOS.xcodeproj/project.pbxproj:1826]: Revert these changes

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsInfoViewController/CallingActionsInfoViewController.swift [hunk 2]
```diff
@@ -123,33 +107,34 @@ class CallingActionsInfoViewController: UIViewController, UICollectionViewDelega
...
         NSLayoutConstraint.activate([
-            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            stackView.topAnchor.constraint(equalTo: view.topAnchor),
-            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            stackView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            stackView.leadingAnchor.constraint(equalTo: view.safeLeadingAnchor),
+            stackView.topAnchor.constraint(equalTo: view.safeTopAnchor),
+            stackView.trailingAnchor.constraint(equalTo: view.safeTrailingAnchor),
+            stackView.bottomAnchor.constraint(equalTo: view.safeBottomAnchor),
+            actionsView.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -32),
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsInfoViewController/CallingActionsInfoViewController.swift:117]:
```suggestion

            actionsView.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -32),
```

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsInfoViewController/CallingActionsInfoViewController.swift [hunk 3]
```diff
@@ -123,33 +107,34 @@ class CallingActionsInfoViewController: UIViewController, UICollectionViewDelega
...
+            actionsViewHeightConstraint,
+            participantsHeaderView.heightAnchor.constraint(equalToConstant: participantsHeaderHeight),
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsInfoViewController/CallingActionsInfoViewController.swift:119]:
```suggestion

            participantsHeaderView.heightAnchor.constraint(equalToConstant: participantsHeaderHeight),
```

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift [hunk 4]
```diff
@@ -53,8 +56,10 @@ class CallingActionsView: UIView {
     private let flipCameraButton = CallingActionButton.flipCameraButton()
     private let endCallButton =  EndCallButton.endCallButton()
     private let handleView = AccessibilityActionView()
+    private let largePickUpButton =  PickUpButton.bigPickUpButton()
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift:59]:
```suggestion
    private let largePickUpButton = PickUpButton.bigPickUpButton()
```

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift [hunk 5]
```diff
@@ -53,8 +56,10 @@ class CallingActionsView: UIView {
...
+    private let largeHangUpButton =  EndCallButton.bigEndCallButton()
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift:60]:
```suggestion
    private let largeHangUpButton = EndCallButton.bigEndCallButton()
```

File: Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (MarcinR) [reply Wire-iOS.xcodeproj/project.pbxproj:1826]: done

**Review by KaterinaWire — APPROVED** [review]
